stack_helpers.sh

```diff
@@ -29,8 +29,21 @@ function update_modules {
 
   cd $to_dir
   $SUDO mkdir -p $name; cd $name
-  $SUDO cp $tmpl_file $version.lua
 
+  # CMAKE_INSTALL_LIBDIR is used by some projects (i.e. lib, lib64)
+  # Detect this when installing the module so the module variables point to the correct path
```

Comment:
can we create that `CMakeLists.txt` file here instead of adding a new file to the repo.  Its purpose is really internal here.